### PR TITLE
fix/v1.4 and v1.5 common revisions to v1.4

### PR DESCRIPTION
**Pull request by nokchalatte**
# Description (개요) 

Add fixes that should be applied to 1.4 and subsequent version(s) (1.5 currently):
- Add missing endpoint docs in engine version 1.4
- Translate tables docs from english to korean for ko version
- Fix incorrect parameter name in tables docs (schema_name -> schema)

## Type of change (작업사항)

Please delete options that are not relevant and check out the type of change.

- [x] REST API docs fix 

## Writing Convention

Make sure to check out the [Writing Convention](https://github.com/smartmind-team/tech-wiki/blob/main/project/thanosql/docs/writing_convention.md). 

## Review Process

Review should be conducted by at least two people.

### CODE REVIEW

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md]:
```suggestion
In order to upload a file, use the below methods to send a file to the Workspace storage.
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md]:
```suggestion
If "db_commit" is set to True and "table_name" and "column_name" are specified, the given file is sent to Workspace storage and the file path is inserted into the specified column of the specified table.
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md]:
```suggestion
In order to delete a file from the Workspace storage, use the below methods.
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md]:
```suggestion
If "db_commit" is set to True and "table_name" and "column_name" are specified, the given file is deleted from Workspace storage, and all rows in the table with the same value as the given file path in the specified column will be deleted.
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md]:
```suggestion
쿼리 로그 목록을 가져옵니다.
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md]:
```suggestion
- `search`: 로그의 쿼리 값 내에서 검색할 문자열.
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md]:
```suggestion
- `search`: The string to search for in the logs' queries
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md]:
```suggestion
- `offset`: 쿼리 로그를 조회를 시작할 기준점 (기본값: 0).
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query.md]:
```suggestion
- `limit`: offset에서 시작하여 조회 할 결과의 개수 (기본값: 100, 최대 100).
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_schema.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_schema.md]:
```suggestion
데이터베이스에서 모든 스키마 목록을 가져옵니다.
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
            - `reference_schema`: `reference_table`을 포함하는 스키마
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
            - `reference_table`: `reference_column`을 포함하는 테이블
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
    _이름 옆에 *가 붙은 위의 구성 요소는 `POST` Table API의 본문(Body)에서는 포함되지 않습니다. 대신 URL에 쿼리 매개변수로 지정됩니다._
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
스키마에서 테이블의 paginated 된 데이터를 가져옵니다.
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
- `offset`: 쿼리 로그를 조회를 시작할 기준점 (기본값: 0).
- `limit`: offset에서 시작하여 조회 할 결과의 개수 (기본값: 100, 최대 100).
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]:
```suggestion
스키마에 있는 테이블의 레코드를 CSV 파일로 가져옵니다.
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md
Comment (stevensjcho) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md]: Note that the other curl examples use {} to show parameters

**Review by stevensjcho — APPROVED** [review]
LGTM 👍. We'll wait for @lazyduo 's review before merging to main.

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (lazyduo) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]: There is absolutely no explanation of what kind of change is being made.
Right before the python/curl code, it is necessary to explain what kind of change it is.
(both en & ko)

**Review by lazyduo — COMMENTED** [review]
리뷰 및 수정 진행했습니다. 코멘트 내용만 추가하고 머지 진행하죠

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md
Comment (stevensjcho) [reply docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md]: 수정 했습니다, 내용 확인 부탁드려요

**Pull request merged**
